src/pps/pps.proto

```diff
@@ -59,12 +59,13 @@ message PipelineJob {
 }
 
 enum PipelineJobState {
-  JOB_STARTING = 0;
-  JOB_RUNNING = 1;
-  JOB_FAILURE = 2;
-  JOB_SUCCESS = 3;
-  JOB_KILLED = 4;
-  JOB_EGRESSING = 5;
+  JOB_CREATED = 0;
```

Comment:
Added a new `JOB_CREATED` job state for when a job has been created by the originating transaction, but before it has been started by a worker.